projects/Uranium.cmake

```diff
@@ -2,10 +2,10 @@ find_package(PythonInterp 3 REQUIRED)
 if(NOT BUILD_OS_WINDOWS)
     # Only ask for Qt5 where it is actually built via cura-build-environment.
     # On Windows we are using PyQt5 to provide our libraries prebuilt.
-    #find_package(Qt5 5.10.0 REQUIRED Core Qml Quick Widgets)
+    #find_package(Qt5 5.15.0 REQUIRED Core Qml Quick Widgets)
 endif()
-find_package(PyQt 5.10 REQUIRED)
-find_package(SciPy 0.17 REQUIRED)
+find_package(PyQt 5.14 REQUIRED)
```

Comment:
Shouldn't this be:
`find_package(PyQt 5.15.2 REQUIRED)`

Reply:
Yes, that would indeed be more correct! I reckon this accidentally slipped in.

Even though it says it needs to find PyQt 5.14, it would still find the 5.15 version (since newer versions are allowed to be found if it doesn't specify `EXACT`).